web/source/dom/domManager.ts

```diff
@@ -1681,9 +1714,14 @@ namespace com.keyman.dom {
           }        
           this.keyman.util.attachDOMEvent(document.body, 'touchstart', (<any>this.keyman).hideOskWhileScrolling, false);
         } else {
+          const _this = this;
           (<any>this.keyman).conditionallyHideOsk = function() {
             // Should not hide OSK if simply closing the language menu (30/4/15)
-            if((<any>keyman).hideOnRelease && !osk['lgList']) osk.hideNow();
+            // or if the focusing timer (setFocusTimer) is still active.
+            if((<any>keyman).hideOnRelease && !osk['lgList'] && !DOMEventHandlers.states.focusing) {
+              _this.touchHandlers.executeBlur(null);
+              osk.hideNow();
+            }
```

Comment:
Context for this change:

https://github.com/keymanapp/keyman/blob/430b636b22efa767b3f94af986ee36757ed24221/web/source/dom/domEventHandlers.ts#L30-L37

Whenever a touch element is focused, there's actually a one-second timer that's supposed to maintain focus on the most recently-touched element.  Probably in case a fat-finger accidentally also touches outside the element when 'releasing' the touch.  So, if a user changes the actively-focused page element too quickly, with the last touch being on the background... KMW actually isn't supposed to hide the OSK.  Oops!  (That's a "today I learned" aspect, actually.)

The simplest way forward to resolving the issues noted in user testing... actually involves fixing that.  (Unless we want to nix the focus-timer.)

Of course, if a `TouchAliasElement`'s active state is maintained in this scenario, it's actually lost DOM focus.  So, that means there's no remaining "blur" event to trigger blur-related stuff for the active element.  Adding the `executeBlur` command here allows this scenario to be handled, preventing a "sticky" active element after the timer completes.